src/MessageContainer.tsx

```diff
@@ -306,7 +310,11 @@ export default class MessageContainer<
 
   renderChatEmpty = () => {
     if (this.props.renderChatEmpty) {
-      return this.props.renderChatEmpty()
+      return (
+        <View style={styles.emptyChatContainer}>
```

Comment:
Hi @iqbalfaisal,
Thanks!
You have to be sure you are not in inverted mode before applying the style.
You should find prop `inverted` here, no?

Reply:
Thanks @xcarpentier 
I have completed this PR. Please review my changes and merge to master.